### DISCUSSION TITLE
Plex requires 20.04 for hardware transcoding of HDR tone-mapping

### DISCUSSION THREAD

**Discussion by thephilipjones** [tteck/Proxmox#90] [General]
Hi, thanks for this awesome collection!

Deploying your Plex LXC script made it way easier to get QuickSync hardware transcoding working than the manual attempts at VM passthrough that I never got right... or so I thought, until I tried a 4K HDR video which doesn't use hardware.

Unfortunately from the links below, it seems the Plex team recommends using Ubuntu 20.04 as a base, and that *no 21.x version* will permit hardware transcoding for HDR tone mapping.

Can you discuss the rationale behind [updating the base Ubuntu image here](https://github.com/tteck/Proxmox/pull/17)?

[Straightforward answer from Plex dev](https://forums.plex.tv/t/lxc-intel-nuc-10-gpu-transcoding-not-working/775655/29)
[Another reference](https://forums.plex.tv/t/anyone-have-been-able-to-hw-transcode-on-an-intel-nuc-11-iris-xe/695381/464)

**Comment by tteck** [comment]
Script changed to use Ubuntu 20.04

**Comment by thephilipjones** [comment]
Need me to submit that as a PR?

**Comment by tteck** [comment]
The script has already been changed from using 21.10 to using 20.04

**Comment by tteck** [comment]
Just create a New Plex Media Server LXC and use the "Copy Data From a Existing Plex Media Server LXC to another Plex Media Server LXC" script

**Comment by thephilipjones** [comment]
Great, thank you! I'm testing it now. (Also shout out for the `pms-copy-data.sh` script making this hopefully mostly painless)
